packages/Testing/PHPUnit/AbstractTestCase.php

```diff
@@ -35,7 +35,7 @@ protected function bootFromConfigFileInfos(array $configFileInfos): void
             $rectorKernel = self::$kernelsByHash[$configsHash];
             self::$currentContainer = $rectorKernel->getContainer();
```

Comment:
I just realized that we already do some cache-re-use in case configs are the same.

I guess this renders the change itself useless. having a non-debug kernel would only make sense in case we would persist the dump-files between ci builds. but this is not a good idea